core/jazz_azure-create-service/index.js

```diff
@@ -0,0 +1,31 @@
+
+const responseObj = require("./components/response.js");
+const CommandMapping = require("./components/CommandMapping.js");
+const fs = require('fs');
+
+module.exports = async () => {
```

Comment:
Will this service (jazz_azure-create-service) live as aws lambda function? 

If not, 
- remove all the jazz components coming from the template - logger, error-handler, config, response etc. This should ideally look like a node module.
- move this to jazz-build-module/azure similar to https://github.com/tmobile/jazz/tree/master/builds/jazz-build-module/apigee
- rename it to jazz-azure-module(?). When I looked at jazz_$foobar, it looked like a core service that needs to be deployed to aws.